tests/console/python310_version_check.pm

```diff
@@ -29,7 +29,7 @@ sub run {
             die("Python default version differs from 3.6");
         }
         my $arch = get_var('ARCH');
-        assert_script_run("SUSEConnect -p sle-module-python3/15.4/$arch");
+        assert_script_run("SUSEConnect -p sle-module-python3/15.5/$arch");
```

Comment:
it's better to use: `add_suseconnect_product('sle-module-python3')`

Reply:
yes, this makes sense, thanks!

See new VR: https://openqa.suse.de/tests/9431427#step/python310_version_check